src/layouts/docs/docs-layout.module.css

```diff
@@ -61,12 +61,18 @@
     */
     & a:global(.__target-h)::before,
     &::before {
+      --permalink-scroll-offset: 24px;
```

Comment:
This just feels like a nice addition to have so that headings aren't right at the top of the viewport when going to a permalink.